server/src/main/java/io/crate/protocols/postgres/types/BitType.java

```diff
@@ -57,6 +58,16 @@ public String type() {
         return Type.BASE.code();
     }
 
+    @Override
+    public Regproc typSend() {
+        return Regproc.of("bit_send");
```

Comment:
Does it make sense to make all of those static vars?

Reply:
I was considering it too, but felt that it's better to create them ad-hoc given that we don't expect constant queries on `pg_type` and they'd then add to the constant memory footprint.